python-threatexchange/threatexchange/exchanges/clients/stopncii/api.py

```diff
@@ -10,7 +10,7 @@
 
 import dacite
 import requests
-from requests.packages.urllib3.util.retry import Retry
+from urllib3.util.retry import Retry
```

Comment:
Pylance was complaining about this, any reason to import from `requests.packages`?

Reply:
Probably earlier confusion from us on what was the correct way to import them. Fine with this change.